.github/workflows/lock.yml

```diff
@@ -0,0 +1,17 @@
+name: Lock Threads
+
+on:
+  schedule:
+    - cron: '0 4 * * *'
```

Comment:
I chose `0 4 * * *` so it should be late evening/night for all of our current core team members.